src/edfi_amt_data_lake/api/api.py

```diff
@@ -48,17 +47,21 @@ def _call(url, token, changeVersionValues) -> list:
 
 # Get JSON from API endpoint and save to file
 def get_all() -> None:
+    import os
+    from multiprocessing import Pool
     toke = get_token()
     changeVersionValues = get_ChangeVersionValues()
-    for endpoint in get_endpoint():
-        url = get_url(endpoint[PATH])
-        data = _call(url, toke, changeVersionValues)
-        endpoint_name = url.split("/")[-1]
-        save_file(JSONFile(endpoint_name), changeVersionValues.newestChangeVersion, data)
-        #Deletes endpoint
-        url_deletes = get_url(f"{url}/deletes")
-        data_deletes = _call(url_deletes, toke, changeVersionValues)
-        save_file(JSONFile(endpoint_name), f"deletes_{changeVersionValues.newestChangeVersion}", data_deletes)
+    max_core = config("MAX_CORE", cast=int) if config("MAX_CORE") else os.cpu_count()
```

Comment:
I recommend a lower default value, like 4, instead of taking all of the available CPUs.